vm/adb/adb.go

```diff
@@ -38,6 +38,9 @@ type Config struct {
 	// This option is enabled by default. Turn it off if your devices
 	// don't have battery service, or it causes problems otherwise.
 	BatteryCheck bool `json:"battery_check"`
+	// set to false if no need adb reboot.
+	// set to true by default.
+	Reboot bool `json:"reboot"`
```

Comment:
We already have a similar parameter in vm/isolated:
https://github.com/google/syzkaller/blob/2af9d324c1bc7d0486122df201cf67590564ae4c/vm/isolated/isolated.go#L34
It does the same, but named differently. Let's keep it consistent and name adb parameter the same way.

I think this will be a better comment:
```
// If this option is set (default), the device is rebooted after each crash.
// Set it to false to disable reboots.
```